modules/browser_object_tabbar.py

```diff
@@ -222,3 +224,14 @@ def scroll_on_all_tabs_menu(self, down=True, pixels=200) -> BasePage:
             self.actions.move_by_offset(0, (sign * pixels))
             self.actions.release()
             self.actions.perform()
+
+    def wait_for_num_tabs(self, num_tabs: int) -> BasePage:
+        """
+        Waits for the driver.window_handles to be updated accordingly with the number of tabs requested
+        """
+        try:
+            WebDriverWait(self.driver, 10).until(
```

Comment:
you don't need a WDW object here, that already exists on the BasePage as `wait`. So, `self.wait.until(...)`

Reply:
fixed!